soh/src/overlays/actors/ovl_En_Cow/z_en_cow.c

```diff
@@ -18,6 +18,9 @@ void func_809E0070(Actor* thisx, GlobalContext* globalCtx);
 
 void func_809DF494(EnCow* this, GlobalContext* globalCtx);
 void func_809DF6BC(EnCow* this, GlobalContext* globalCtx);
+struct CowInfo EnCow_GetInfo(EnCow* this, GlobalContext* globalCtx);
```

Comment:
this should just be `CowInfo EnCow_GetInfo` since we aren't defining the struct here, just using it

Reply:
Build doesn't seem to like me removing struct here

Reply:
try moving the struct definition up? might need to use typedef

Reply:
yeah, that was c++ brain kicking in

in c to use the struct without the keyword struct you need to typedef it

i'm fine with keeping this as is (i'd still prefer it the other way but i'm not going to let that hold back merging this)